docs/user_guide/installation_and_version_guide/v0.7.1.rst

```diff
@@ -0,0 +1,25 @@
+v0.7.1
```

Comment:
I added this here because of [this comment](https://github.com/fairlearn/fairlearn/pull/997#pullrequestreview-878163541). Considering the recent changes about including/not including this PR in v.8 milestones, maybe it should be changed to v0.8.0.rst or removed completely? Wdyt @hildeweerts @romanlutz @adrinjalali

Reply:
This file has been renamed in `main`: https://github.com/fairlearn/fairlearn/blob/main/docs/user_guide/installation_and_version_guide/v0.8.0.rst